tests/Doctrine/Tests/DBAL/Platforms/MySQL57PlatformTest.php

```diff
@@ -78,7 +78,7 @@ protected function getQuotedAlterTableRenameIndexInSchemaSQL()
     protected function getGeneratesAlterTableRenameIndexUsedByForeignKeySQL()
     {
         return array(
-            'ALTER TABLE mytable RENAME INDEX idx_foo TO idx_foo_renamed',
+            'ALTER TABLE mytable RENAME INDEX `idx-foo` TO `idx-foo-renamed`',// Index name with special character in name (needs quotation on some platforms, e.g. Sqlite).
```

Comment:
@Ocramius   Today, i am usign doctrine and i have a field that name was datetime and the doctrine crash. the quota is necesary on the field name, same bug with a field named to.